backoffice/src/Controllers/DroitController.cs

```diff
@@ -249,7 +249,27 @@ private async Task<DroitForm> PopulateDroitFormAsync(DroitForm form)
 
         return form;
     }
-    
+
+    public async Task<IActionResult> Export(string query)
+    {
+        //This needs to use the search form

```

Comment:
This needs to use the same search as the droit form, then get all the results and output them. 

The fields to be outputted are Everything* 


- All droit fields, Wreck name, salvor name, some other stuff? TBC